.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,191 @@
+name: Nightly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "0 0 * * 1-5"
+
+jobs:
+  fedora:
+    name: Linux Fedora (Release, GCC, cxx11)
+    runs-on: ubuntu-latest
+    container: dynawo/dynawo-distribution-cxx11:latest
+    strategy:
+      fail-fast: false
+    env:
+      DYNAWO_INSTALL_OPENMODELICA: /opt/OpenModelica/Install
+      DYNAWO_SRC_OPENMODELICA: /opt/OpenModelica/Source
+      DYNAWO_RESULTS_SHOW: "false"
+      DYNAWO_BUILD_TYPE: Release
+      DYNAWO_FORCE_CXX11_ABI: "true"
+      DYNAWO_CXX11_ENABLED: YES
+      DYNAWO_NB_PROCESSORS_USED: 2
+      GH_API: ${{ github.api_url }}/repos/${{ github.repository }}
+      GH_API_DYNAWO: ${{ github.api_url }}/repos/${{ github.repository_owner }}/dynawo
+      CURL_JSON: 'curl --silent --header "Content-Type": "application/json"'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch Dynawo
+        run: |
+          source .github/workflows/tools.sh
+
+          download_asset() {
```

Comment:
could you add this function in the ressource file ?

Reply:
and is it different from download_file ?

Reply:
yes, `download_asset` is different from `download_file`: `download_asset` "knows" things about how the assets in a release are organized (which URL has to be used to download them). `download_file` is generic, it is just fetching a file from an URL.

I left `download_asset` directly in the actions workflow because it is specific, but there is no problem in moving it to the resource file, I will do it.

Reply:
Done. Should I squash again ?

Reply:
yes it is better

Reply:
Done